### DISCUSSION TITLE
Should the react peer dependency be marked optional?

### DISCUSSION THREAD

**Discussion by triptu** [pmndrs/zustand#407] [General] [answered]
So I was trying to use vanilla zustand directly in a non react project. There are no issues but yarn install does give me an "unmet peer dependency" warning. Should we use the [peerdependenciesmeta](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta) field to mark the dependency optional? I can submit a PR for this.

**Comment by dai-shi** [comment, marked as answer]
Yeah, that makes sense for vanilla users. Please open a PR.

**Reply by triptu** [reply]
Thanks, I have created a PR - https://github.com/pmndrs/zustand/pull/408
